src/core/Metavariable.ml

```diff
@@ -34,6 +34,12 @@ let logger = Logging.get_logger [ __MODULE__ ]
  *)
 type mvar = string [@@deriving show, eq, hash]
 
+module MvarSet = Set.Make (struct
```

Comment:
You can also use 't Set_.t' (Set_.ml is a polymorphic Set in commons/), or SSet.t (defined in Common.mli I think)